__tests__/app.test.js

```diff
@@ -14,26 +14,7 @@ afterAll(() => {
   
 
 describe("app", () => {
-

```

Comment:
There are a lot of changes to files here, try to be consistent and make as fact structural changes to stuff as possible as you are going as the pull request gets very messy